src/Raven.Server/Utils/Cpu/CpuUsageCalculator.cs

```diff
@@ -104,7 +114,15 @@ private double CalculateProcessCpuUsage(ProcessInfo currentInfo, double machineC
                 processCpuUsage = Math.Min(processCpuUsage, machineCpuUsage);
             }
 
-            return Math.Min(100, processCpuUsage);
+            if (processCpuUsage < 0 && Logger.IsInfoEnabled)
+            {
+                Logger.Info($"processCpuUsage == {processCpuUsage}, OS: {RuntimeInformation.OSDescription}");
+            }
+
```

Comment:
Is that needed?

Reply:
processCpuUsage < 0 is not supposed to happen at this point, but if there is an issue, we won't be able to detect it because the value is clamped between 0 and 100